examples/js-minifier/src/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+   * Add two numbers.
+   * 
+   * @param {number} first The first number.
+   * @param {number} second The second number.
+   * @returns {number} The sum of the two numbers.
+**/
```

Comment:
JSDoc dosesn't look right to me. You'll need to remove the leading spaces a bit. 

Something like these:
https://github.com/kabirbaidhya/webpack-file-preprocessor-plugin/blob/a363762b97f2e56579de9a71b62974d8b739fabe/src/util.js#L21-L47

Please refer to this for more info - https://eslint.org/docs/rules/valid-jsdoc

Reply:
Done 👍